### ISSUE TITLE
Cursor placed before the hotkey query

### ISSUE THREAD

**Issue by yuwen-ye** [Flow-Launcher/Flow.Launcher#1295]
**Describe the bug/issue**
The cursor is placed before the hotkey query. 

**To Reproduce**
1. Set a Hotkey and a custom Query (in my case, I set the `ctrl + space` to trigger the window walker)
2. Press the hotkey `ctrl + space`
3. The cursor was placed before the query `w`. 

**Screenshots**
![image](https://user-images.githubusercontent.com/66440048/180675879-dd09e7be-1b2a-4fd5-bcbc-496eac7d1a02.png)

**Your System**
```
Windows build number: (run "ver" at a command prompt)
Microsoft Windows [Version 10.0.19043.1826]

Flow Launcher version: (Settings => About)
1.9.4 
```
**Flow Launcher Error Log**

**Comment by taooceros** [comment]
You can check the latest dev version in the discussion page. It should has been fixed.

**Comment by yuwen-ye** [comment]
@taooceros Thank you for letting me know that. Are you planning to bring this bug fix to the stable version in the near future?

**Comment by taooceros** [comment]
> @taooceros Thank you for letting me know that!
> 
> Are you planning to bring this bug fix to the stable version in the near future?

for sure, it will be released with 1.10 once we finish up everything else.
